src/pages/using-spark/components/button.mdx

```diff
@@ -18,29 +19,42 @@ An interactive element that represents an action or choice.
 
 ### Usage
 
-Use Buttons to represent the main actions on a page.
+Use buttons to represent what type of actions the users can take on a page and to create a visual hierarchy between different calls to action.
+
+> As a general rule, use no more than two primary call-to-action buttons ([Primary Button](#primary-button)) per page.
 
 Examples of Button usage:
-- To trigger a new UI element to appear on the page.
+- To trigger action.
 - To submit a form.
 - To begin a new task.
 - To navigate to the next step in a process.
+- To confirm completion or to exit out of a flow.
+
+If the action will take take the user to a new page or to an external site, consider using a [Link](/using-spark/components/link) instead.
 
 ### Guidelines
 
-- Button Text
-  - Use precise, task-specific action verbs like: “Add”, “Remove”, “Read Details”, and “Submit”.
-  - Avoid ambiguous words like “Yes”, “No”, “OK”, and “Click Here”.
-  - Use title case (capitalize the first letter each word)
-  - Use less than 20 characters.
-  - Never wrap to a second line.
-- Button width should not exceed 280px.
-- Do not use buttons in paragraphs or within a lines of text. Use [Link](/using-spark/components/link) instead.
-- Use Button variants to create action hierarchy within a page. Variant names imply their placement within that hierarchy.
-  - [Primary Button](#primary-button)
-  - [Secondary Button](#secondary-button)
-  - [Tertiary Button](#tertiary-button)
-  - [Quaternary Button](#quaternary-button)
+#### How to Write Effective Button Text
+- Button text should set clear expectation of what will happen on click.
+- Always use title case (capitalize the first letter each word).
+- Use task-specific action verbs like: “Add”, “Remove”, “Read Details”, and “Submit”.
+- Use less than 20 characters.
+- Use compact words. For example, instead of “Submit Your Information Now” use “Submit”.
+- Use first-person pronouns. Use “Customize My Loan,” instead of “Customize Your Loan.”
+- Do not use ampersands (&).
+- Do not wrap button text into a second line; it reduces legibility, and starts looking less like an interactable element.
+
+#### Button Styling
+- Text should be center aligned.
+- Buttons should have 16px of padding on the top and bottom, and 24px on the left and right sides.
+- Width should be between 48x and 280px.
+- Grouped buttons should be 16px away from each other.
+- Do not use buttons in paragraphs or within lines of text. Use [Link](/using-spark/components/link) instead.
+
+#### Full Width at Smaller Viewports
+At small viewport sizes, a Button may be full width.
+- Stacked Buttons should have the primary action above secondary actions.
+-	Use full width Buttons in containers that take up 1/3 of the grid or smaller.
```

Comment:
Extra space here.

Reply:
Oh weird! must be some weird ASCII special character that did that because on the code it looks fine. Must have been artifacts from copying and pasting from Word.  Fixed it in the code by deleting and adding a space.